task/kaniko/0.4/kaniko.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: kaniko
+  labels:
+    app.kubernetes.io/version: "0.4"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/tags: image-build
+    tekton.dev/displayName: "Build and upload container image using Kaniko"
```

Comment:
I guess displayName can be more precise like `kaniko`. WDYT?

Reply:
I used the the codecov task as an example of how this field should look:

https://github.com/tektoncd/catalog/blob/main/task/codecov/0.1/codecov.yaml#L11

From that I guessed that the display name was supposed to be a short sentence describing the task. If that isn't correct I will happily change it.

Reply:
It can be anything. It can be one word or it's depends on the author. I have no issues.